chart/yugaware/templates/application.yaml

```
@@ -0,0 +1,85 @@
+      * **YugabyteDB-as-a-Service on Kubernetes** - Deploy your own YugabyteDB-as-a-Service running on any Kubernetes infrastructure, Yugabyte Platform provides central management for multiple database clusters and Includes 24/7 production support.  
+      * **Monitoring & Alerting** - Comprehensive monitoring & alerting without any configuration.  
+      * **Scheduled Backups** - Use Yugabyte Platform to perform a regular scheduled backup of a full YugabyteDB universe or selected tables.
+
+      ### Getting Started
+      Here are a few resources for getting started with Yugabyte platform
+      * [Quick start guide for Yugabyte Platform](https://docs.yugabyte.com/latest/yugabyte-platform/deploy/) - Install Yugabyte platform.
+      * [Quick Start for YugabyteDB](http://docs.yugabyte.com/latest/quick-start/) - install, create a local cluster and read/write from YugaByte DB.  
+      * [Explore core features](https://docs.yugabyte.com/latest/explore/) - automatic sharding & re-balancing, linear scalability, fault tolerance, tunable reads etc.  
+      * [Ecosystem integrations](https://docs.yugabyte.com/latest/develop/ecosystem-integrations/) - integrations with Apache Kafka/KSQL, Apache Spark, JanusGraph, KairosDB, Presto and more.  
+      * [Real world apps](https://docs.yugabyte.com/latest/develop/realworld-apps/) - sample real-world, end-to-end applications built using YugaByte DB.  
+
+      ### Help
+      Cannot find what you are looking for? Have a question? We love to hear from you - [please file a GitHub issue](https://github.com/YugaByte/yugabyte-db/issues) or talk to us live at [Slack](https://www.yugabyte.com/slack).
+    notes: |-
+      # Access Yugabyte Platform UI
+
+      ## Through GKE UI
+      Find the IP from Yugabyte Platform GKE App.
+
+      ## Through kubectl
+      ```
+      kubectl get svc -n "${NAMESPACE}" \
+        "{{ .Release.Name }}-yugaware-ui"
+      ```
+
+      Follow the [link](https://docs.yugabyte.com/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/kubernetes/) to know more about to deploy the YugaByteDB universe.
+
+    maintainers:
+      - name: YugaByte
+        url: http://www.yugabyte.com/
+    keywords:
+      - database
+      - SQL
+      - yugabyte
+    links:
+      - description: 'Yugabyte Platform'
+        url: http://www.yugabyte.com/README.md
+  info:
+    - name: Yugabyte Platform UI
+      type: Reference
+      valueFrom:
+        serviceRef:
+          name: "{{ .Release.Name }}-yugaware-ui"
+    - name: Application namespace
+      value: "{{ .Release.Namespace }}"
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: "{{ .Release.Name }}"
+  addOwnerRef: true
+  componentKinds:
+  - group: 'v1'
+    kind: PersistentVolumeClaim
+  - group: ''
+    kind: Secret
+  - group: ''
+    kind: Service
+  - group: ''
+    kind: StatefulSet
+  - group: 'v1beta1'
+    kind: PodDisruptionBudget
+  - group: ''
+    kind: ConfigMap
```

```suggestion
  componentKinds:
  - group: ''
    kind: PersistentVolumeClaim
  - group: ''
    kind: Secret
  - group: ''
    kind: Service
  - group: ''
    kind: StatefulSet
  - group: 'policy'
    kind: PodDisruptionBudget
  - group: ''
    kind: ConfigMap
```

https://v1-18.docs.kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#poddisruptionbudget-v1beta1-policy